pom.xml

```
@@ -4,7 +4,8 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.40</version>
+    <version>4.44</version>
+    <relativePath />
```

BTW
```suggestion
    <relativePath/>
```
The extra space is superfluous in XML; but traditional browser HTML parsers required it for XHTML IIRC; and when using `maven-release-plugin` the crappy XML rewriting tool always reinserts it. JEP-229 avoids the problem by not mechanically editing the version POM.

---

The vast majority of our existing plugin POMs have the space so I was just following existing precedent. SortPom allows both conventions via the `spaceBeforeCloseEmptyElement` configuration option, which we've set to true in core to keep with existing precedent. Maybe someday when we run SortPom on more POMs that would be a good opportunity to clean this up globally.